### PR TITLE
Fixed netfx System.CodeDom.Tests on non English Windows

### PR DESCRIPTION
**Pull request by AlexRadch**
See https://github.com/dotnet/corefx/issues/28149 and https://github.com/dotnet/corefx/issues/28136

### CODE REVIEW

File: src/System.CodeDom/tests/CSharpCodeGenerationTests.cs [hunk 1]
```diff
@@ -614,125 +615,160 @@ public void Arrays_WithinArrays_Empty_NonPrimitiveTypes()
...
-            field1.CustomAttributes.Add(new CodeAttributeDeclaration("System.Xml.Serialization.XmlElementAttribute"));
-            field1.InitExpression = new CodePrimitiveExpression("hi!");
-            class1.Members.Add(field1);
+                var attrs = cu.AssemblyCustomAttributes;
+                attrs.Add(new CodeAttributeDeclaration("System.Reflection.AssemblyTitle",
+                    new CodeAttributeArgument(new CodePrimitiveExpression("MyAssembly"))));
+                attrs.Add(new CodeAttributeDeclaration("System.Reflection.AssemblyVersion",
+                    new CodeAttributeArgument(new CodePrimitiveExpression("1.0.6.2"))));
+                attrs.Add(new CodeAttributeDeclaration("System.CLSCompliantAttribute",
+                    new CodeAttributeArgument(new CodePrimitiveExpression(false))));
```
Comment (stephentoub) [inline src/System.CodeDom/tests/CSharpCodeGenerationTests.cs]: Can you please revert all of this newline / whitespace formatting that was done?

Comment (AlexRadch) [reply src/System.CodeDom/tests/CSharpCodeGenerationTests.cs]: I will.

**Comment by pjanotti** [comment]
Hi @AlexRadch thanks for finding the issue and already proposing a fix.

**Comment by pjanotti** [comment]
@tarekgh the code here is single-threaded do we need to go the route of RemoteInvoke or is it good enough to set the culture of the current thread? I just want to avoid the remote invoke if possible just to keep the tests faster

**Comment by AlexRadch** [comment]
@pjanotti xUnit can run tests in different threads as I know.

**Comment by pjanotti** [comment]
Yes, but a test method without await/async (or same native api forking execution) will be entirely in the same thread. Even if it blocks it will resume execution on the same thread. If this is acceptable it will be basically the same as your change but instead of setting the global culture it sets only the one of the current thread.

**Comment by pjanotti** [comment]
Ah @tarekgh just told me what I was missing: setting thread culture in UWP affects the whole process. Currently we don't ship this package for UWP, however, to avoid having it as a special case let's go with the traditional route of RemoteInfoke. One may wonder about CodeDom in UWP but in theory it could be used to generate source files, since JIT is not allowed. That said the current implementation is not generating code anyway...

**Comment by pjanotti** [comment]
>             CultureInfo.CurrentUICulture = CultureInfo.InvariantCulture;

Missing RemoteInvoke here.

---
Refers to: src/System.CodeDom/tests/CSharpCodeGenerationTests.cs:1456 in 351d812. [](commit_id = 351d812554715bd8eae65007b1ac0883d6f8a84b, deletion_comment = False)

**Comment by pjanotti** [comment]
>             var cuic = CultureInfo.CurrentUICulture;

Missing RemoteInvoke

---
Refers to: src/System.CodeDom/tests/CSharpCodeGenerationTests.cs:2581 in 351d812. [](commit_id = 351d812554715bd8eae65007b1ac0883d6f8a84b, deletion_comment = False)

**Comment by pjanotti** [comment]
>             var cuic = CultureInfo.CurrentUICulture;

Missing RemoteInvoke

---
Refers to: src/System.CodeDom/tests/VBCodeGenerationTests.cs:1405 in 351d812. [](commit_id = 351d812554715bd8eae65007b1ac0883d6f8a84b, deletion_comment = False)

**Comment by pjanotti** [comment]
>             CultureInfo.CurrentUICulture = CultureInfo.InvariantCulture;

Missing RemoteInvoke

---
Refers to: src/System.CodeDom/tests/VBCodeGenerationTests.cs:2373 in 351d812. [](commit_id = 351d812554715bd8eae65007b1ac0883d6f8a84b, deletion_comment = False)

**Comment by AlexRadch** [comment]
@dotnet-bot test Windows x86 Release Build

**Comment by pjanotti** [comment]
Test failures are not related to this PR.

**Comment by pjanotti** [comment]
Thanks @AlexRadch for this contribution.

**Pull request merged**